src/Cli/dotnet/commands/dotnet-new/README.md

```diff
@@ -0,0 +1,25 @@
+## `dotnet new`
+
+This is a home for `dotnet new` command.
+
+The issues for `dotnet new` be opened in dotnet/sdk repo with label [`area: dotnet new`](https://github.com/dotnet/sdk/labels/area%3A%20dotnet%20new).
```

Comment:
Instead of suggesting to add a label, consider adding Issue Templates for `dotnet new` behavior and/or template content specifically.

Reply:
This is a nice idea, but I don't have permissions to do it. @marcpopMSFT could you please help with it? Thank you.